code/modules/jobs/job_types/ai.dm

```diff
@@ -15,6 +15,7 @@
 	exp_type_department = EXP_TYPE_SILICON
 	display_order = JOB_DISPLAY_ORDER_AI
 	var/do_special_check = TRUE
+	job_skills = list(SKILL_DEBUG) // silicons and AI get debug to bypass skill checks until there is a need to give AI or cyborg specific skills
```

Comment:
i think it would be great to have different skills loaded into individual modules